frontend/src/components/UserTables/StudentsTable.tsx

```diff
@@ -66,6 +66,15 @@ const StudentsTable = ({ searchName }: studentTableProps) => {
   const formatDate = (date: string): string =>
     moment(date).format('MM/DD/YYYY');
 
+  const filterStudents = (filter: boolean) =>
```

Comment:
The name `filter` doesn't really describe the function it's providing anymore, now that we're reducing the filtering when the checkbox is on. Let's give it a more appropriate name.

Reply:
This comment is talking moreso about the state name from the `useState` call earlier in the file, not this parameter in particular.

Reply:
Got it. I am thinking the the function works the `filter useState `name should be changed to something along the lines of `showInactive` as that is the main purpose of the boolean

Reply:
Yep, that's a great name